File: app/src/main/java/com/hlibrary/action/entity/BasePhoneInfo.java
package com.hlibrary.action.entity;

import android.graphics.Bitmap;

import java.io.Serializable;

/**
 * @author 林文辉
 * @version v 1.0.0
 * @since 2015-01-27
 */
public class BasePhoneInfo implements Serializable {

    public static final String PHOTO_INFOS = "PHOTO_INFOS";

    /**
     *
     */
    private static final long serialVersionUID = -7184597475809248127L;

    private Bitmap bitmap;

    public Bitmap getBitmap() {
        return bitmap;
    }

    public void setBitmap(Bitmap bitmap) {
        this.bitmap = bitmap;
    }


}

File: app/src/main/java/com/hlibrary/entity/BasePhoneInfo.kt
package com.hlibrary.entity

import android.graphics.Bitmap
import java.io.Serializable

/**
 * @author 林文辉
 * @version v 1.0.0
 * @since 2015-01-27
 */
class BasePhoneInfo : Serializable {
    var bitmap: Bitmap? = null

    companion object {
        private const val serialVersionUID = -7184597475809248127L
    }
}